app_dart/lib/src/foundation/utils.dart

```diff
@@ -122,3 +144,43 @@ Future<void> insertBigquery(
     log.warning('Failed to add build status to BigQuery: $ApiRequestError');
   }
 }
+
+/// Returns luci try builders for difference repo by calling corresponding config getters.
+Future<List<Map<String, dynamic>>> getLuciTryBuilders(String repo, Config config) async {
+  List<Map<String, dynamic>> luciTryBuilders;
+  switch (repo) {
+    case 'flutter':
+      luciTryBuilders = await config.flutterLuciTryBuilders;
+      break;
+    case 'engine':
+      luciTryBuilders = await config.engineLuciTryBuilders;
+      break;
+    case 'cocoon':
+      luciTryBuilders = await config.cocoonLuciTryBuilders;
+      break;
+    case 'packages':
+      luciTryBuilders = await config.packagesLuciTryBuilders;
+      break;
+  }
```

Comment:
You may want to rm this switch case and pass  slug.name directly to the method getting the configs.

Reply:
please refer the earlier comment.